features/bevragen/persoon-beperkt/verificatie/dev/fields-gba.feature

```diff
@@ -0,0 +1,44 @@
+#language: nl
+
+Functionaliteit: GbaPersoonBeperkt: verificatie velden vragen met fields
+    Achtergrond:
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+      | geslachtsnaam (02.40) | geboortedatum (03.10) |
+      | Vries                 | 19781103              |
+      En de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                             | waarde               |
+      | datum verificatie (71.10)        | 20020701             |
+      | omschrijving verificatie (71.20) | bewijs nationaliteit |
+
+  Rule: verificatie van de persoonsgegevens wordt bij elke vraag teruggegeven
+
+    Scenario: persoonsgegevens zijn geverifieerd en verificatie wordt niet gevraagd
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam          | waarde                              |
+      | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+      | geslachtsnaam | Vries                               |
+      | geboortedatum | 1978-11-03                          |
+      | fields        | geslacht                            |
+      Dan heeft de response een persoon met de volgende 'verificatie' gegevens
+      | naam         | waarde               |
+      | datum        | 20020701             |
+      | omschrijving | bewijs nationaliteit |
+
```

Comment:
Ik probeer het te begrijpen en heb eea teruggezocht. Lees in de mail van Frans van 20/8 : "Maar in de specifieke situatie dat je alleen gegevens opvraagt die niet door enige ABO worden bijgehouden, is die automatisch meeverstrekte verificatie eigenlijk niet aan de orde; daar heb je gelijk in. Het zou beter zijn als die verificatie in zo’n geval niet meeverstrekt zou worden."
Ik vraag mij nu af welke categorieen precies wel en niet door een abo worden bijgehouden... Hoort geslacht daar wel of niet bij?

Reply:
Besproken in overleg en besloten de functionaliteit zo eenvoudig mogelijk te houden. er zijn geen user stories bekend dat dit gegeven ergens voor gebruikt wordt door onze afnemers. We beschrijven dus vooralsnog geen logica voor wanneer wel en wanneer niet leveren verificatie in relatie tot wat er verder gevraagd of geleverd wordt. Verificatie werkt dus hetzelfde als geheimhoudingPersoonsgegevens en opschortingBijhouding